src/gtimelog/main.py

```diff
@@ -1440,7 +1465,8 @@ def populate_log(self):
                     total += item.duration
                 prev = item.start
         elif self.detail_level == 'grouped':
-            work, slack = window.grouped_entries()
+            work, slack = window.grouped_entries(
+                sorted_by=self._get_log_order_key())
```

Comment:
I'm somewhat surprised that this setting takes effect only in grouped view, and not also in summary view.

Reply:
To be honest, I got lazy, and it would have been another completely different implementation, because the structures are so different, but I can probably do it.

Reply:
It's not a merge blocker.  If you don't do it, I think I will.

Reply:
OK, thanks.